app/uk/gov/hmrc/cgtpropertydisposalsstubs/controllers/ReturnController.scala

```diff
@@ -31,25 +33,43 @@ import uk.gov.hmrc.cgtpropertydisposalsstubs.util.GenUtils.sample
 import uk.gov.hmrc.cgtpropertydisposalsstubs.util.Logging
 import uk.gov.hmrc.play.bootstrap.controller.BackendController
 
+import scala.io.Source
 import scala.util.Try
 
 @Singleton
 class ReturnController @Inject() (cc: ControllerComponents) extends BackendController(cc) with Logging {
 
   def submitReturn(cgtReferenceNumber: String): Action[JsValue] =
     Action(parse.json) { request =>
-      val submittedReturn: JsResult[(BigDecimal, LocalDate)] = for {
+
+      import Version7._
```

Comment:
isn't the version of the schema we have v4?

Reply:
good catch, fixing

Reply:
fixed